assets/prometheus-k8s/prometheus.yaml

```diff
@@ -195,6 +195,7 @@ spec:
   - prometheus-k8s-thanos-sidecar-tls
   - kube-rbac-proxy
   - metrics-client-certs
+  - federate-client-certs
```

Comment:
hmm I expect the federate-client-certs secret to be mounted into the telemeter client deployment?

Reply:
right